io.catenax.reuse_certificate/2.1.0/ReuseCertificate.ttl

```diff
@@ -0,0 +1,47 @@
+#######################################################################
+# Copyright (c) 2023, 2024 SAP SE
+# Copyright (c) 2023, 2024 Bayerische Motoren Werke Aktiengesellschaft
+# Copyright (c) 2023, 2024 LRP Autorecycling Leipzig GmbH
+# Copyright (c) 2023, 2024 Fraunhofer-Gesellschaft zur Förderung der angewandten Forschung e.V.
+# Copyright (c) 2023, 2024 ZF Friedrichshafen AG
+# Copyright (c) 2023, 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This work is made available under the terms of the
+# Creative Commons Attribution 4.0 International (CC-BY-4.0) license,
+# which is available at
+# https://creativecommons.org/licenses/by/4.0/legalcode.
+#
+# SPDX-License-Identifier: CC-BY-4.0
+#######################################################################
+
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.1.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.1.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.reuse_certificate:2.1.0#> .
```

Comment:
it's better to change this model version to 3.0.0 instead 2.1.0. As this is non compatible with previous version and also would be aligned with shared model versions.